core/types/block.go

```diff
@@ -275,10 +273,10 @@ func CopyHeader(h *Header) *Header {
 		cpy.ShardState = make([]byte, len(h.ShardState))
 		copy(cpy.ShardState, h.ShardState)
 	}
-	if len(h.CrossLinks) > 0 {
```

Comment:
If we comment out crosslinks, we have to do fork later to bring it back. probably just keep it as placeholder

Reply:
We will have to do fork either way. Now, all these are just wasting spaces.